app/gui2/e2e/selectingNodes.spec.ts

```diff
@@ -23,14 +30,12 @@ test('Selecting nodes by click', async ({ page }) => {
   await locate.graphNodeIcon(node1).click({ modifiers: ['Shift'] })
   await expect(node1).toBeSelected()
   await expect(node2).toBeSelected()
+  await deselectAll()
```

Comment:
Here also: I want to test that even having two nodes selected, when clicking one, the other is deselected.